packages/client/src/views/RecordAudit/ActionDetailsModal.tsx

```diff
@@ -63,23 +63,9 @@ function retrieveUniqueComments(
     return []
   }
 
-  if (previousHistoryItemIndex === -1) {
-    return actionDetailsData.comments
-      .map((comment) => comment?.comment)
-      .map((comment) => ({ comment }))
-  }
-
-  const comments: IDynamicValues[] = []
-  actionDetailsData.comments.forEach((item, index) => {
-    if (
-      (histories[previousHistoryItemIndex].comments || [])[index]?.comment !==
-      item?.comment
-    ) {
-      comments.push({ comment: item?.comment })
-    }
-  })
+  const comment = actionDetailsData.comments[1]?.comment
 
-  return comments
```

Comment:
It's probably better to handle the correction comment separately as changing this function would affect the action modal for other actions